decidim-core/app/jobs/decidim/resend_confirmation_instructions_job.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module Decidim
+  class ResendConfirmationInstructionsJob < ApplicationJob
+    queue_as :default
+
+    def perform(user)
+      user.send_reconfirmation_instructions
```

Comment:
I noticed that the email in the "Welcome ..." message is different than the email where the message was sent to:
![Email confirmation](https://user-images.githubusercontent.com/864340/162142600-39402754-2e54-4dbc-b330-c43fe54866f2.png)

Is it possible to match these?

I also think that when changing the email address, the welcome message is not necessary. It's nice to have there after the initial registration but it seems a bit weird when changing the email. So, one way to fix this would be to take out the "Welcome" line completely for reconfirmations.

Reply:
We could leave it out when unconfirmed email isn't present: [`659ef56` (#8996)](https://github.com/decidim/decidim/pull/8996/commits/659ef560b94f5e3d2cad649bc14b0c70fec4176f#diff-46e8f47c806165cd8f362cab5991ecfe43ad9c1ace6df066331dff5abe864cbeR1)